UefiCpuPkg/PiSmmCpuDxeSmm/X64/PageTbl.c

```diff
@@ -962,13 +962,10 @@ SmiPFHandler (
     }
 
     if (mCpuSmmRestrictedMemoryAccess && IsSmmCommBufferForbiddenAddress (PFAddress)) {
-      DumpCpuContext (InterruptType, SystemContext);

```

Comment:
This change the order of:
1. DumpCpuContext
2. DumpModuleInfoByIp

Normally, we think this is more reason we see the #PF dump code:
![image](https://github.com/user-attachments/assets/53faa7ac-639e-4caa-8916-1f5f00940842)

How about move the DumpModuleInfoByIp into below else logic?

Reply:
moved the DumpModuleInfoByIp into below else logic